icu4c/source/layoutex/cmemory.h

```diff
@@ -0,0 +1 @@
+../common/cmemory.h
```

Comment:
symlink. may not work everywhere.

Reply:
Do these symlinks replace some kind of manual step of copying files?
Shouldn't these be handled by setting a `-I` include folder path?